### PR TITLE
return correct OpenAPI yaml for json values

### PR DESCRIPTION
**Pull request by dkarwacki**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -330,7 +330,7 @@ lazy val json4s: ProjectMatrix = (projectMatrix in file("json/json4s"))
     name := "tapir-json-json4s",
     libraryDependencies ++= Seq(
       "org.json4s" %%% "json4s-core" % Versions.json4s,
-      "org.json4s" %%% "json4s-jackson" % Versions.json4s % Test,
+      "org.json4s" %%% "json4s-jackson" % Versions.json4s,
```
Comment (adamw) [inline build.sbt:333]: for sure we want to keep the jackson dependency in the `Test` scope :)

File: docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlTest.scala [hunk 2]
```diff
@@ -908,6 +908,70 @@ class VerifyYamlTest extends AnyFunSuite with Matchers {
     actualYamlNoIndent shouldBe expectedYaml
   }
 
+  test("should match the expected yaml for circe json value") {
```
Comment (adamw) [inline docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlTest.scala:911]: these are quite heavy tests for a small detail. Maybe we could simply test how schemas look like in the json integrations?

**Comment by adamw** [comment]
Could you maybe link/reference the issues from PRs? Will be easier to navigate :)

File: build.sbt (hunk 1 above)
Comment (dkarwacki) [reply build.sbt:333]: ah, sure, it should be directly copied into test dependencies of tapir-openapi-docs..

File: docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlTest.scala (hunk 2 above)
Comment (dkarwacki) [reply docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlTest.scala:911]: ah, yes, that's good idea

Comment (dkarwacki) [reply docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlTest.scala]: corrected

**Pull request merged**